proto/process/agent.proto

```diff
@@ -904,6 +904,10 @@ message DNSStats {
 	map<uint32, uint32> dnsCountByRcode = 4;
 }
 
+message HTTPAggregations {
```

Comment:
I'm not super happy with these names (HTTPAggregations, HTTPStats etc.) Suggestions are welcome